README.md

```diff
@@ -420,6 +433,26 @@ Style: 0 / 0
 Total: 100 / 100
 ```
 
+We can target a specific submission ID through `--target-submission`.
+Use the `autograder.cli.courses.assignments.submissions.fetch.user.history` command to see a student's previous submission IDs and scores.
```

Comment:
Instead of listing the full command here, link to the section in this document (`#getting-a-history-of-all-past-submissions`).

I would also soften up the "Use" a bit, since there are other ways to get the ID.
So something like "You can use" would be better.